src/stylesheets/elements/form-controls/_checkbox-and-radio.scss

```diff
@@ -55,6 +55,10 @@
       background: color($input-bg-color);
       box-shadow: 0 0 0 units($theme-input-select-border-width)
         adjust-color(color($input-text-color), $alpha: $input-border-alpha);
+      @media (forced-colors: active) {
+        outline: 2px solid transparent;
+        outline-offset: units(2px);
```

Comment:
Let's use the `2px` value consistently, instead of one using `2px` and the other `units()`.

Reply:
Resolved in https://github.com/uswds/uswds/pull/4509/commits/085c6da44e9876d51b4c550cd8883758dfdf9980